spring-cloud-starter-task-spark-cluster/README.adoc

```diff
@@ -0,0 +1,23 @@
+//tag::ref-doc[]
+= Spark Cluster Task
+
+This task is intended to launch a Spark application from a task. The task submits the Spark application for execution in a Spark cluster. This task is appropriate for a deployments where any file references can be resolved to a shared location.
```

Comment:
First sentence should read: This task is intended to launch a Spark application. Will fix on merge